packages/editor-ui/src/components/Node/NodeCreator/Modes/NodesMode.vue

```diff
@@ -139,6 +139,13 @@ function onSelected(item: INodeCreateElement) {
 			searchItems: mergedNodes,
 		});
 	}
+
+	if (item.type === 'link') {
+		window.open(item.properties.url, '_blank');
```

Comment:
Should this use `item.properties.newTab`?

Reply:
IMO we should always open NodeCreator links in a new tab